tests/consumers/test_bookmark_save_rmq.py

```
@@ -64,6 +68,37 @@ def should_not_call(**kwargs):
+    """필수 필드 누락 시 처리 테스트"""
+    class DummyMessage:
+        def __init__(self, body):
+            self.body = body
+        def process(self):
+            class Ctx:
+                async def __aenter__(inner):
+                    return self
+                async def __aexit__(inner, exc_type, exc, tb):
+                    pass
+            return Ctx()
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)